integration/fixtures/docker/simple.toml

```diff
@@ -9,8 +9,7 @@
 logLevel = "DEBUG"
 
 [docker]
-
 # It's dynamagic !
 endpoint = "{{.DockerHost}}"
-
 domain = "docker.localhost"
+watch = true
```

Comment:
@emilevauge I would prefer if we do a `simple-watch.toml` for the test that involve `watch` to be true (or the other way aronud). wdyt ? :stuck_out_tongue_closed_eyes: